### PR TITLE
filter bad effects from modifiers instead of hard coding

### PR DESCRIPTION
**Pull request by Rinn**

### CODE REVIEW

**Comment by Rinn** [comment]
hold up a bit on this one I'm also adding Always Fumble to mafia rn

File: packages/garbo/src/mood.ts [hunk 1]
```diff
@@ -227,23 +228,29 @@ export function useBuffExtenders(): void {
...
-const textAlteringEffects = $effects`Can Has Cyborger, Dis Abled, Haiku State of Mind, Just the Best Anapests, O Hai!, Robocamo`;
-export const teleportEffects = $effects`Teleportitis, Feeling Lost, Funday!`;
-const otherwiseBadEffects = $effects`Temporary Blindness`;
+const damageEffects = Effect.all().filter(
+  (x) =>
+    numericModifier(x, "Thorns") ||
+    numericModifier(x, "Sporadic Thorns") ||
+    numericModifier(x, "Damage Aura") ||
+    numericModifier(x, "Sporadic Damage Aura"),
+);
```
Comment (horrible-little-slime) [inline packages/garbo/src/mood.ts:237]:
```suggestion
const damageEffects = Effect.all().filter(
  (x) =>["Thorns", "Sporadic Thorns", "Damage Aura", "Sporadic Damage Aura"].some((modifier) => numericModifier(x, modifier) > 0));
```

Comment (gausie) [reply packages/garbo/src/mood.ts]: I would pull out the array of strings from being defined within the filter.

**Comment by Rinn** [comment]
okay should be good now

**Review by horrible-little-slime — APPROVED** [review]

**Pull request merged**